docs/howto/openstack/cinder/retype-volumes.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Changing the type of a volume ("retyping") is an offline operation that requires detaching the volume from its server, and setting its new type.
+---
+# Changing a volume's type
+
+You may occasionally need to change the type of a volume.
+This may be due to a volume type being phased out on {{brand}}'s part (necessitating data migration), or you might want to enable or disable [encryption](encrypted-volumes.md) on a volume.
+
+{{page.meta.description}}
+The resulting down time may be quite substantial, particularly for large volumes.
+As such, when you need to retype volumes, you should plan ahead well in advance.
+
+## Prerequisites
+
+In order to retype volumes, you must use the OpenStack CLI.
+Make sure you have [enabled it](../../getting-started/enable-openstack-cli.md).
+
```

Comment:
OK, this is just me, but I feel this is more "natural":

"In order to retype volumes you must use the OpenStack CLI, so make sure you have it [enabled](../../getting-started/enable-openstack-cli.md)"

Reply:
Don't mind either way. Will change.